tests/nonlinear_channel_flow_tractions_Newton_Stokes_no_line_search.prm

```diff
@@ -0,0 +1,148 @@
+# Like the poiseuille_2d.prm test and based on the nonlinear channel 
+# flow benchmark. This is used to test the velocity boundary conditions 
+# of the Newton Stokes solver.
+
+set Dimension = 2
+set CFL number                             = 1.0
+set Maximum time step                      = 1
+set End time                               = 4
+set Start time                             = 0
+set Adiabatic surface temperature          = 0
+set Surface pressure                       = 0
+set Use years in output instead of seconds = false  # default: true
+set Nonlinear solver scheme = iterated Advection and Newton Stokes
+set Max nonlinear iterations = 45 
+set Nonlinear solver tolerance = 2.5e-10
+set Additional shared libraries = ../benchmarks/nonlinear_channel_flow/libsimple_nonlinear.so
```

Comment:
Remove this line, it is confusing (and not used) in a test. If this is only a slightly modified copy of a benchmark or another test, consider using a `include $ASPECT_SOURCE_DIR/tests/visualization_strain_rate_tensor.prm` statement, as long as you only need to add a few lines afterwards.